__tests__/index.spec.ts

```diff
@@ -23,12 +29,17 @@ const ChildComponent = defineComponent({
             class="child-component--loading">
             Source: {{ source }}
         </div>
+        <div
+            class="child-component--state">
```

Comment:
**issue:** I think you forgot to add a test here right @FloEdelmann?

Reply:
Actually I wanted to leave it like that. When the instance somehow isn't passed correctly, just rendering this throws an error. But if you want, I can also add a test case.

Reply:
Yes test case would be nice